src/Tribe/Template.php

```diff
@@ -402,27 +402,27 @@ protected function get_template_public_path( $base ) {
 	 * @return array
 	 */
 	protected function get_template_path_list() {
-		$folders = array();
+		$folders = [];
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: 252a06f0f2826c46dc439f00668e642d</sub>